main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -128,21 +128,25 @@ object Monomorpher {
     /**
```

Comment:
I think this class has some documentation at the top that should now be updated?

Reply:
yes, have yet to do a documentation pass